packages/terra-clinical-result/docs/ClinicalResultBloodPressure.md

```diff
@@ -0,0 +1,95 @@
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+```
+$ npm install terra-clinical-result
+```
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Component Features
+
+* [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
+* [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
+* [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
+* [LTR/RTL Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#ltr--rtl-support)
+ 
+<!-- Uncomment supported features.
+* [Internationalization Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#internationalization-i18n-support)
+-->
+
+## Usage Guide
+
+The blood pressure result is constructed similarly to the single clinical result by providing structured object with a specific construction that follows a similar pattern to the [HL7 FHIR Observation Blood Pressure](https://www.hl7.org/fhir/observation-example-bloodpressure.html) example.
+```jsx
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: PropTypes.shape({
+    /**
+     *  Systolic Result for blood pressure.
+     */
+    systolic: observationPropShape,
+    /**
+     *  Diastolic Result for blood pressure.
+     */
+    diastolic: observationPropShape,
+  }),
+```
+
+
+An example of a single blood pressure result value:
+```jsx
+import React from 'react';
+import { ClinicalResultBloodPressure } from 'terra-clinical-result/lib/index';
+
+const bloodpressureResultValue = {
+  id: '111',
+  systolic: {
+    eventId: '111.1',
+    result: {
+      value: '180',
+      unit: 'mmHg',
+    },
+    interpretation: 'CRITICAL',
+    type: 'BLOODPRESSURE',
+    isModified: true,
+    hasComment: true,
+    isUnverified: true,
+    conceptDisplay: 'Blood Pressure Systolic',
+    datetimeDisplay: 'Nov 23, 2019 13:31:31',
+  },
+  diastolic: {
+    eventId: '111.2',
+    result: {
+      value: '60',
+      unit: 'mmHg',
+    },
+    interpretation: 'LOW',
+    type: 'BLOODPRESSURE',
+    isModified: true,
+    hasComment: true,
+    isUnverified: true,
+    conceptDisplay: 'Blood Pressure Diastolic',
+    datetimeDisplay: 'Nov 23, 2019 13:31:31',
+  }
+};
+
+export default () => <ClinicalResultBloodPressure resultData={bloodpressureResultValue} hideUnit isTruncated />;
+
+```
+
+## Clinical Result Props
+
+|Prop Name|Type|Is Required|Default Value|Description|
```

Comment:
Lets use the props table loader for this section

Reply:
Switched here https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13#diff-0748fd62687d573e95ea9081ee65da9eL86